ci/verify_exported_targets/CMakeLists.txt

```diff
@@ -71,11 +68,78 @@ add_test_case(t120 storage_client)
 add_test_case(t130 googleapis-c++::spanner_client)
 add_test_case(t140 spanner_client)
 
-add_test_case(t200 PkgConfig::google_cloud_cpp_common)
-add_test_case(t210 PkgConfig::google_cloud_cpp_grpc_utils)
-add_test_case(t220 PkgConfig::bigtable_client)
-add_test_case(t230 PkgConfig::firestore_client
+# TODO(#5726) - DO NOT CHANGE THESE MODULE NAMES These module names are expected
+# to remain usable until we have retired them.  This is a test for backwards
+# compatibility. If you need to change the test, there is good chance you are
+# breaking something.
+set(pkg_config_modules
+    google_cloud_cpp_common google_cloud_cpp_grpc_utils bigtable_client
+    pubsub_client spanner_client storage_client)
+foreach (module ${pkg_config_modules})
+    pkg_check_modules(${module} IMPORTED_TARGET REQUIRED ${module})
+    add_test_case(test_pc_${module} PkgConfig::${module})
+endforeach ()
+
+# This one is special because it does not depend on google_cloud_cpp_common.
+pkg_check_modules(firestore_client IMPORTED_TARGET REQUIRED firestore_client)
```

Comment:
(no change necessary) Given Firestore's experimental status, I'm not sure we need to do anything special to maintain backward compat for it. However, since we're doing these checks for the other libraries, this Firestore check likely won't add any extra constraints on us. So, I'd be happy to remove these Firestore checks, or keep them, but I did want to comment to record that there was a question about them here.

Reply:
Ack.